tests/test_solido.py

```diff
@@ -523,16 +588,58 @@ def deposit(lamports: int, expect_created_token_account: bool = False) -> None:
     solido_program_id,
     keypair_path=maintainer.keypair_path,
 )
+
+del result['StakeDeposit']['stake_account']
 expected_result = {
     'StakeDeposit': {
-        'validator_vote_account': validator_vote_account.pubkey,
-        # We have 1.0 SOL from the true deposit, and 1.0 donated.
-        'amount_lamports': int(2.0e9),
+        'validator_vote_account': validator_1.vote_account.pubkey,
+        'amount_lamports': 2_050_250_000,
```

Comment:
How did we get to this number? It looks a bit arbitrary to me, can you add a comment about why we should get this particular amount? (E.g. deposited X and Y = Z and we divide that over n validators.)